lib/excision_web/live/decision_site_live/show.html.heex

```diff
@@ -56,14 +56,19 @@
   <:item title="Most Recent Decision"><%= @most_recent_decision %></:item>
   <:item title="# Classifiers"><%= @num_classifiers %></:item>
   <:item title="Active Classifier">
-    <.link
-      patch={
-        ~p"/decision_sites/#{@decision_site}/classifiers/#{@decision_site.active_classifier}"
-      }
-      class="hover:underline"
-    >
-      <%= @decision_site.active_classifier.name %>
-    </.link>
+    <div class="flex items-center">
+      <.link
+        patch={
+          ~p"/decision_sites/#{@decision_site}/classifiers/#{@decision_site.active_classifier}"
+        }
+        class="hover:underline"
+      >
+        <%= @decision_site.active_classifier.name %>
+      </.link>
+      <%= if Excision.Excisions.is_default_classifier?(@decision_site.active_classifier) do %>
+        <ExcisionWeb.Components.TooltipComponent.tooltip text='When a Decision Site is created, an associated "Passthrough" entity is also automatically created, which replicates the same behavior as making a function call to a model provider, with the added benefit that decisions coming from the LLM are recorded in your DecisionBox database.   Those decisions can now be inspected and labeled, establishing a baseline for accuracy before you replace it with a newly created, purpose-built classifier which can be trained through labeling for even greater accuracy.  Note that a Passthrough cannot be trained.' />
+      <% end %>
```

Comment:
If this message is exactly the same as the other tooltip, please share the messages